### PR TITLE
Add missing plugin for publishing

### PR DESCRIPTION
**Pull request by majk-p**

### CODE REVIEW

File: build.sbt
Comment (majk-p) [inline build.sbt:50]: I'll remove this before merge, once we confirm that publishing works

File: project/plugins.sbt
Comment (majk-p) [inline project/plugins.sbt:13]: Those are removed in favor of `sbt-typelevel-ci-release` that includes them both

**Review by zainab-ali — APPROVED** [review]

**Comment by majk-p** [comment]
Blocked by https://github.com/typelevel/sbt-typelevel/pull/722

**Pull request merged**
